packages/common/src/content/compose.ts

```diff
@@ -694,15 +713,8 @@ export const composeContent = (
         // TODO: groups?
       };
     },
-    // TODO: do we want to add dataset extent logic as a getter here?
```

Comment:
Is the dataset extent logic referenced in this comment the same checks I put in `determineExtent`?

Reply:
thanks for checking, but I'm not sure. I think this was an out dated comment.